typings/styled-system.d.ts

```diff
@@ -321,8 +325,23 @@ declare namespace StyledSystem {
   export function responsiveStyle(args: LowLevelStyleFunctionArguments): any
 
   export function pseudoStyle(args: LowLevelStyleFunctionArguments): any
-}
 
-declare module "styled-system" {
-  export = StyledSystem
+  /**
+   * Artsy
+   *
+   * WIP / FIXME / Additions
+   */
+
+  export type BorderValue = string | number
+
+  export interface BorderProps {
+    border?: BorderValue
+  }
+
+  // FIXME: Should be able to use ColorProps above
+  export interface ColorProps {
+    color?: ColorValue
+    bg?: ColorValue
+    backgroundColor?: ColorValue
+  }
```

Comment:
@zephraph - I added this section at the bottom of the file for WIP things. Unless you have a better idea we should put all of our additions here so that we can keep track / iterate.

Reply:
Ahhh, makes sense. Yeah, I like it. 👍